eos_data_distribution/SimpleStore/__init__.py

```diff
@@ -27,7 +27,7 @@
 
 class Producer(object):
 
-    def __init__(self, base=None, prefix='/',
+    def __init__(self, base='.', prefix='/',
```

Comment:
Commit c1405228 basically supersedes this one entirely, so this one can be dropped.